compose/cli/docker_client.py

```diff
@@ -9,27 +8,9 @@ def docker_client():
     Returns a docker-py client configured using environment variables
     according to the same logic as the official Docker client.
     """
-    cert_path = os.environ.get('DOCKER_CERT_PATH', '')
-    if cert_path == '':
-        cert_path = os.path.join(os.environ.get('HOME', ''), '.docker')
```

Comment:
Well spotted. Looks like that logic does indeed still exist in Docker:

- [read DOCKER_CERT_PATH](https://github.com/docker/docker/blob/master/docker/flags.go#L28)
- [set it to cliconfig.ConfigDir() if empty](https://github.com/docker/docker/blob/master/docker/flags.go#L76)
- [cliconfig sets it to $HOME/.docker if empty](https://github.com/docker/docker/blob/master/cliconfig/config.go#L33)

So I think we should add it to docker-py first - this PR isn't urgent.